### PR TITLE
Updated changelog

### PR DESCRIPTION
**Pull request by MartijnR**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+- `orx:max-pixels` bind attribute to transform uploaded images.
+- Submission chapter.
+- `orx:auto-send` submission element attribute.
+- `orx:auto-delete` submission element attribute.
+- `<odk:rank>` form control.
+- [`digest`](https://opendatakit.github.io/xforms-spec/#fn:digest) function.
+
+#### Changed
+- Corrected argument order in [`jr:choice-name`](https://opendatakit.github.io/xforms-spec/#fn:jr:choice-name) function.
+- Corrected Psuedo-number generator used in [`randomize`](https://opendatakit.github.io/xforms-spec/#fn:randomize) function.
```
Comment (lognaturel) [inline CHANGELOG.md]: psuedo -> pseudo

Comment (MartijnR) [reply CHANGELOG.md]: Ah yes, that's my thing. I never know. :(

**Comment by MartijnR** [comment]
Done!

File: CHANGELOG.md (hunk 1 above)
Comment (lognaturel) [reply CHANGELOG.md]: English spelling makes no sense. (is that Greek or something?)

**Review by lognaturel — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lognaturel) [reply CHANGELOG.md]: Or you could always go with German "ersatz". Ersatz-number generator has a nice ring to it.

Comment (lognaturel) [reply CHANGELOG.md]: Wait a minute. It should be pseudo-random number generator, no? It's the randomness that's ersatz, not the number!

Comment (MartijnR) [reply CHANGELOG.md]: Yes, oh my. This is turning out to be the hardest PR to get approved! :) 👍

Comment (MartijnR) [reply CHANGELOG.md]: Oh, it was merged anyway. Woohoo!

Comment (lognaturel) [reply CHANGELOG.md]: Well, I already merged it. But see, my goofing around had value. ;)

Comment (lognaturel) [reply CHANGELOG.md]: Heh. We can keep that between us and see if anyone notices.

**Pull request merged**